test/testtokenize.cpp

```diff
@@ -8087,7 +8087,7 @@ class TestTokenizer : public TestFixture {
         ASSERT_THROW(tokenizeAndStringify("void foo() { for_chain( if (!done) done = 1); }"), InternalError);
         ASSERT_THROW(tokenizeAndStringify("void foo() { for_chain( a, b, if (!done) done = 1); }"), InternalError);
 
-        ASSERT_THROW_EQUALS(tokenizeAndStringify("void f() { if (retval==){} }"), InternalError, "syntax error: == )");
+        ASSERT_THROW_EQUALS(tokenizeAndStringify("void f() { if (retval==){} }"), InternalError, "Syntax Error: AST broken, 'if' doesn't have two operands.");
```

Comment:
Personally I like the `Syntax error: == )` message better. It points out the problem better.